### DISCUSSION TITLE
load data spacy

### DISCUSSION THREAD

**Discussion by sk-cmd** [explosion/spaCy#7266] [Help: Coding & Implementations]
how can use my personal data instead of data downloaded by spacy ? because I want to use data from poorly endowed resources which are not downloadable by spacy. Do you can help me please!!!
Note ! i should not use     de_model = spacy.load('de') and en_model = spacy.load('en')

**Comment by polm** [comment]
Not entirely sure I follow what you're trying to do, but maybe you want to work with data in a language spaCy doesn't provide a model for? The documentation about creating a custom language might be useful. 

https://spacy.io/usage/linguistic-features#language-subclass

You could also try using the [multilanguage "xx" models](https://spacy.io/models/xx).

If that's not what you had in mind, maybe you could paste a short example of the kind of data you're working with?
